data-analysis/re-creation.ipynb

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
what is this file for?

Reply:
this is the percent by demographics analysis - I named it re-creation because it goes off of the U of Vermont paper analysis. I can change the name to something more recognizeable